src/utils/get-lybsin-podcasts.js

```diff
@@ -0,0 +1,51 @@
+// Fetches podcast info from https://mongodb.libsyn.com/rss
+
+const RSS_URL = `https://mongodb.libsyn.com/rss`;
+
+export const simplifyPodcast = podcast => {
+    const podcastJSON = {
+        title: podcast['title'],
+        publishDate: podcast['pubDate'],
+        summary: podcast['itunes:summary'],
+        url: podcast['enclosure']['@_url'],
+    };
+    return podcastJSON;
+};
+
+export const parsePodcasts = podcastXML => {
+    var parser = require('fast-xml-parser');
+
+    var options = {
+        ignoreAttributes: false,
+        parseAttributeValue: true,
```

Comment:
We can also set the `attributeNamePrefix` to '' to get rid of that odd `@_` for url.